gsa/src/web/graphql/__mocks__/alerts.js

```diff
@@ -120,3 +120,30 @@ const mockAlerts = {
 
 export const createGetAlertsQueryMock = () =>
   createGenericQueryMock(GET_ALERTS, {alerts: mockAlerts});
+
+const createAlertResult = {
+  createAlert: {
+    id: '12345',
+    status: 200,
+  },
+};
+
+export const createAlertInput = {
+  name: 'foo',
+  event: 'NEW_SECINFO_ARRIVED',
+  condition: 'ALWAYS',
+  method: 'HTTP_GET',
+  methodData: {
+    URL: 'yourdomain.com',
+    composer_include_notes: 1,
+    composer_include_overrides: 0,
```

Comment:
Should not be send via the api. At least not as methodData. We need to think about the api changes for the content composer sometime in future.